lib/loaders/__tests__/elastic-resource-loader.js

```diff
@@ -1,4 +1,4 @@
-const elasticsearch         = require('elasticsearch');
+const elasticsearch         = require('@elastic/elasticsearch');
```

Comment:
It looks like the tests are missing?

Reply:
As discussed, I initially inherited this with empty tests and skipped implementing any in the name of expediency. I've gone back and implemented them, and in the process found (and fixed) some bugs, so that ended up working out.